recipes/svgpathtools/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "svgpathtools" %}
+{% set version = "1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/teseoch/svgpathtools/archive/{{ version }}.tar.gz
```

Comment:
Is the original at mathandy/svgpathtools? It might be confusing to build off your fork.  Has the upstream been abandoned?

Reply:
I dont know. I forked it because it had bugs.

Reply:
Hmm we shouldn't be building your fork here under the same name.  If you changed the package name and re-released it we could package it.  Have you tried getting your fixes in upstream?